hack/lib/util/misc.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash
+#
+# This library holds miscellaneous utility functions. If there begin to be groups of functions in this
+# file that share intent or are thematically similar, they should be split into their own files.
+
+# os::util::describe_return_code describes an exit code
+#
+# Globals:
+#  - START_TIME
```

Comment:
It would make sense for callers (i.e., `os::util::trap::exit_handler`) to pass this value as a parameter instead of using the global.  Making it a parameter would also avoid one potential pitfall: setting the global variable, but forgetting to export it, which would mean it would be unavailable here because `os::util::trap::exit_handler` invokes `os::util::describe_return_code` in a subshell.